mgmt/rest/v2/plugin.go

```diff
@@ -183,40 +211,44 @@ func (s *apiV2) loadPlugin(w http.ResponseWriter, r *http.Request, _ httprouter.
 				}
 			}
 
-			// A little sanity checking for files being passed into the API server.
-			// First file passed in should be the plugin. If the first file is a signature
-			// file, an error is returned. The signature file should be the second
-			// file passed to the API server. If the second file does not have the ".asc"
-			// extension, an error is returned.
-			// If we loop around more than twice before receiving io.EOF, then
-			// an error is returned.
-
-			switch {
-			case i == 0:
-				if filepath.Ext(p.FileName()) == ".asc" {
-					e := errors.New("Error: first file passed to load plugin api can not be signature file")
-					Write(400, FromError(e), w)
+			switch p.FormName() {
+			case "ca_certs":
+				fn, err := createTempFile(b, TLSCACertsPrefix)
```

Comment:
@candysmurf, I looked more thoroughly on this PR, and I do not think so that creating "temp" file is a good approach. What needs to be done here is only checking if provided path exists - so why you decided to add 
 [createTempFile](https://github.com/intelsdi-x/snap/pull/1703/files#diff-d96666069f58edcf865ae5d038e1ce95R299) function instead of straightforward do sth like that:  `if _, err := os.Stat(certPath); os.IsNotExist(err)` ?

My preference is avoid creating temp file if they are not needed. Please consider following what is done in V1 - https://github.com/intelsdi-x/snap/blob/master/mgmt/rest/v1/plugin.go#L144